test/DartTest.ts

```diff
@@ -107,6 +108,24 @@ describe('transpile to dart', () => {
     });
   });
 
+  describe('enums', () => {
+    it('should support basic enum declaration', () => {
+      expectTranslate('enum Color { Red, Green, Blue }')
+          .to.equal(' enum Color { Red , Green , Blue }');
+    });
+    it('does not support empty enum', () => {
+      chai.expect(() => translateSource('enum Empty { }'))
+          .to.throw('empty enums are not supported');
+    });
+    it('does not support enum with initializer', () => {
```

Comment:
Could we address that in a separate issue / PR? Support or not of `const enum`s is IMHO debatable (in particular since there is no equivalent feature in Dart.